homework/web_scrapping1.py

```diff
@@ -0,0 +1,38 @@
+import requests
+from bs4 import BeautifulSoup 
+import sqlite3
```

Comment:
@sampleharvgithub Any reason for using sqlite database instead of using postgresql(mentioned in aim of homework)?

Reply:
I tried to include the database into the webscrapping python code. And it works fine, hence i didn't use postgresql.

Reply:
@sampleharvgithub Firstly, sqlite isn't scalable and has several other limitations. Hence it's not used in real world projects. Also postgresql basics were covered in workshop-1 which is why we expect postgresql to be used.